pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -35,7 +40,19 @@ in stdenv.mkDerivation {
     inherit url hash sha256;
   };
 
-  nativeBuildInputs = [ addOpenGLRunpath ];
+  nativeBuildInputs = [
+    autoAddOpenGLRunpathHook
```

Comment:
I didn't know there way such a hook. Is this new?

Reply:
Yes I created it in the redist packages PR.